compiler/aarch64/codegen/OpBinary.cpp

```diff
@@ -886,6 +887,8 @@ const OMR::ARM64::InstOpCode::OpCodeBinaryEntry OMR::ARM64::InstOpCode::binaryEn
 		0x4E405800,	/* UZP2      	vuzp2_8h */
 		0x4E805800,	/* UZP2      	vuzp2_4s */
 		0x4EC05800,	/* UZP2      	vuzp2_2d */
+	/* Vector extract */
+		0x6E000000, /* EXT       	vext16b	 */
```

Comment:
Please use a tab character between `,` and the comment in this line.

Reply:
Fixed it.